src/app/core/core.module.ts

```diff
@@ -0,0 +1,39 @@
+import { NgModule, Optional, SkipSelf } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { NavModule } from './nav/nav.module';
+import { HomeModule } from '../home/home.module';
+import { ProfileModule } from '../profile/profile.module';
+import { SignUpModule } from '../sign-up/sign-up.module';
+import { LoginModule } from '../login/login.module';
+import { NgAwsCognitoModule } from '../ng-aws-cognito';
```

Comment:
We should import the NgAwsCognitoModule from 'ng-aws-cognito' instead of using a relative path since thats how developers using the library will do it